src/network/core_utils.rs

```diff
@@ -799,11 +799,11 @@ impl CoreUtils {
                 }
 
                 if ipv6_enabled {
-                    // Disable duplicate address detection if ipv6 enabled
+                    // Enable optimistic_dad on interface if ipv6 is enabled
                     // Do not accept Router Advertisements if ipv6 is enabled
-                    let br_accept_dad = format!("/proc/sys/net/ipv6/conf/{}/accept_dad", ifname);
+                    let br_optimistic_dad = format!("/proc/sys/net/ipv6/conf/{}/optimistic_dad", ifname);
                     let br_accept_ra = format!("net/ipv6/conf/{}/accept_ra", ifname);
```

Comment:
Question not sure about this but do we need to enable `Router Advertisements` i.e `accept_ra` ?

Reply:
I don't know. I think this was copied from cni.